fuzzers/run_fuzzer.py

```diff
@@ -333,52 +333,70 @@ def should_run_submake(make_flags):
 
 
 def main(argv):
-    fuzzers = os.path.abspath(os.path.dirname(__file__))
+    fuzzers_dir = os.path.abspath(os.path.dirname(__file__))
 
     parser = argparse.ArgumentParser(description=__doc__)
     parser.add_argument("fuzzer", help="fuzzer to run")
+    parser.add_argument(
```

Comment:
Probably shouldn't do this?  Flaky fuzzers should be fixed at the fuzzer level.

Reply:
I generally agree, until all the fuzzers have been fixed this is a work around. I will log a bug to remove it. I am also hoping to mark the fuzzer as flaky in the test results output.

Reply:
At least default it to no retries.  Then maybe have CI run it will higher retry count, but other users would see the flakyness?